linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -29,9 +29,23 @@ import EmitterNames._
 
 private[emitter] object CoreJSLib {
 
-  def build(jsGen: JSGen, globalKnowledge: GlobalKnowledge): WithGlobals[Tree] =
+  def build(jsGen: JSGen, globalKnowledge: GlobalKnowledge): WithGlobals[Lib] =
     new CoreJSLibBuilder(jsGen)(globalKnowledge).build()
 
+  /** A fully built CoreJSLib
+   *
+   *  @param definitions The bulk of the CoreJSLib.
+   *      Definitions that do not depend on any other Scala.js emitted code
+   *      (n.b. RuntimeLong). These must be available to all Scala.js emitted
```

Comment:
Did you mean `e.g.` instead of `n.b.`? Or did you mean "notably"?

Reply:
I did mean notably / nota bene.

Reply:
Hum but "nota bene" doesn't mean notably. It means "Pay attention to:", whereas "notably" can be rephrased as "in particular".

Reply:
Ah, ah :) So I guess "notably" is most appropriate here? (IMHO it's going a bit far to call `RuntimeLong` an example :P).